src/applications/vaos/utils/selectors.js

```diff
@@ -1,3 +1,9 @@
+export function selectConfirmedAppointment(state, id) {
+  return (
+    state.appointments?.confirmed?.find(appt => appt.uniqueId === id) || null
```

Comment:
may not be necessary if `confirmed` is always going to be an array but I think this can optionally chain the function call too. 

`state.appointments?.confirmed?.find?.(appt => appt.uniqueId === id) || null`